### DISCUSSION TITLE
LinkedIn, Tik Tok, Pinterest?

### DISCUSSION THREAD

**Discussion by akalexander-gh** [chatwoot/chatwoot#5586] [Feature Requests]
Hi there,

I'm not a developer by any means, but was referred to GitHub by support, to request features.

Our company corresponds frequently on LinkedIn messages, occasionally on Pinterest messages, and increasingly so on Tik Tok. 

Are there any plans for these channels to be incorporated into Chatwoot? 

We are not currently customers/users of Chatwoot but are currently searching for a solution, and hoping Chatwoot will be said solution. Thanks!

**Comment by sojan-official** [comment]
@akalexander-gh  at the moment these channels are not a priority because as far as I know TikTok, Pinterest, etc don't have open APIs for us to pull in the messages to Chatwoot.
